docs/ops/sequence/GRUSequence_1.md

```diff
@@ -0,0 +1,136 @@
+## GRUSequence <a name="GRUSequence"></a> {#openvino_docs_ops_sequence_GRUSequence_1}
```

Comment:
Please, update the "docs/ops/opset4.md" with GRUSequence_1 and RNNSequence_1

Reply:
updated